README.md

```diff
@@ -12,19 +12,26 @@ Set up Gradle 8.7 or later, and make `build.gradle` like below.
 
 ```
 plugins {
-    id "org.embulk.runset" version "0.1.0"  // Just apply this Gradle plugin.
+    id "org.embulk.runset" version "0.2.0"  // Just apply this Gradle plugin.
 }
 
 repositories {
     mavenCentral()
 }
 
 installEmbulkRunSet {
-    embulkHome file("path/to/embulk-home")  // Set your Embulk home directory (absolute path) to install the Embulk plugins.
+    // Set your Embulk home directory (absolute path) to install the Embulk plugins and "embulk.properties".
+    embulkHome file("/path/to/your-embulk-home")
 
     artifact "org.embulk:embulk-input-postgresql:0.13.2"
     artifact group: "org.embulk", name: "embulk-input-s3", version: "0.6.0"
+
+    // It downloads jruby-complete-9.1.15.0.jar, and set the "jruby" Embulk System Property in "embulk.properties".
```

Comment:
Minor Adding the location of download JRuby file may useful. what do you think?

Reply:
Adding some explanation in README? Or as a kind of feature?

Reply:
It meant in README.

```java
// JRuby install in the xxx directory
```

Reply:
The path would be written in `embulk.properties`. I do not want to make it "documented" because documenting it would chain our future changes.

Reply:
I see, Thanks!